src/ccl/charts/charts-container.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+
+import { ChartService } from '../services/chart.service';
+import { Chart } from './chart.component';
+
+/*
+ * Charts container component
+ * Holds all the charts
+ */
+
+@Component({
+  selector: 'charts',
+  directives: [Chart],
+  styleUrls: [
+    './chart.style.css',
+  ],
+  template: `<div class="chart" *ngFor="let chart of chartList">
+                <chart [indicator]="chart" [chartData]="chartData"></chart>
+            </div>`
+})
+
+export class Charts extends OnInit {
+
+    private chartList: Array<String>;
```

Comment:
Here's another place where typescript could be advantageous. I would eventually expect this component to only require an array of Chart objects, returned from the chartService, rather than having to pull the chartList and the chartData separately.

Also ok with deferring this, but its worth thinking about because its still easy ATM to change the public interfaces of these components.